packages/medusa/src/api/routes/store/customers/index.ts

```diff
@@ -34,7 +34,11 @@ export default (app, container) => {
   route.get("/me", middlewares.wrap(require("./get-customer").default))
   route.post("/me", middlewares.wrap(require("./update-customer").default))
 
-  route.get("/me/orders", middlewares.wrap(require("./list-orders").default))
+  route.get(
+    "/me/orders",
+    middlewares.normalizeQuery(),
```

Comment:
suggestion: we could defenatly integrate the new middleware approach here and in the handler. wdyt?